ts/store/reducers/notifications/installation.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Notifications installation reducer
+ */
+
+import uuid from "uuid/v4";
+
+import { Action } from "../../../actions/types";
+import { GlobalState } from "../../../reducers/types";
+import { NOTIFICATIONS_INSTALLATION_TOKEN_UPDATE } from "../../actions/constants";
+
+export type InstallationState = Readonly<{
+  uuid: string;
+  token: string | null;
```

Comment:
excluding React stuff, `undefined` should be preferred to `null` for optional values, see [this explanation](https://palantir.github.io/tslint/rules/no-null-keyword/)